providers/logzio/logzio_provider.go

```diff
@@ -26,8 +26,8 @@ import (
 
 type LogzioProvider struct {
 	terraform_utils.Provider
-	token   string
-	baseURL string
+	api_token string // this must match the Config in the provider
```

Comment:
Need to be apiToken, and baseURL

Reply:
The intention was to avoid having to make changes to the logzio terraform provider which might have knock on issues.

If the string doesn't match, like using `baseURL` there `coerceValue` function returns `unexpected attribute baseURL`

Reply:
in `GetConfig` function  it's can be `api_token` as string in map, but golang variables name can be  `baseURL` and `apiToken`. LogzioProvider get settings from `GetConfig`

Reply:
I misunderstood what you were saying, sorry. 

Corrected now